### ISSUE TITLE
Spaces for indentation, not so good

### ISSUE THREAD

**Issue by meh** [rubocop-hq/ruby-style-guide#49]
I suppose few people will be with me on this, but meh.

Recently I converted from being a space guy to a tab guy, the thing is simple, tabs for **indentation** are awesome, because the next guy can choose the indentation space, there's people that like 2 spaces, there's people that like 4, and there's even people that like 8, why forcing them to see everything in 2 spaces?

2 spaces indentation:
![2 spaces indentation](http://i.imgur.com/SsmtZ.png)

8 spaces indentation:
![8 spaces indentation](http://i.imgur.com/rj1dU.png)

The underlined part after the tab (it starts at that grey dot) are spaces, this means that the real look of the code won't change, only the indentation.

In short, tabs for indentation, spaces for aligning. If your code looks different when you change indentation size, you're doing it wrong.

/discuss

**Comment by lsmolic** [comment]
I know this was closed a long time ago, but incase it comes up again, this was very impactful to read and I hope others take away from it 

https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/

**Comment by mariocha** [comment]
I totally agree with Ismolic.
Furthermore, even for non-impaired readers. It is much easier to grasp at a glance 2 tabs + 2 spaces then 10 spaces.
IMHO, the real goal of the guide is that code is EASY (and fast) to read and THUS accepted by the community.
